sqlserver/assets/configuration/spec.yaml

```
@@ -39,6 +39,15 @@ files:
       description: Password for the Datadog-SQL server check user. It will be ignored if using Windows authentication.
       value:
         type: string
+    - name: server_version
+      description: |
+        Server version year of sqlserver the agent will connect to. 
+        Important for validating connection string attributes for older sqlserver versions.
+        
+        This is required if connecting to a SQLServer instance older than 2014.
+      value:
+        type: string
+        example: "2019"
```

Sorry one more small nit!

```suggestion
        example: "2014"
```

---

oh duh, ty